src/utils/j1ql-validator.ts

```diff
@@ -0,0 +1,301 @@
+import { J1qlService } from '../client/services/j1ql-service.js';
+
+interface ValidationResult {
```

Comment:
I don't think this is being used either, is it? Do we have any value in keeping it?